semcore/icon/svg/color/Confluence/l.svg

```diff
@@ -1 +1,18 @@
-<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M2.337 17.267c-.215.35-.457.758-.646 1.083a.663.663 0 0 0 .222.9l4.307 2.651a.663.663 0 0 0 .917-.225c.17-.288.391-.663.633-1.064 1.706-2.815 3.425-2.471 6.516-.993l4.27 2.03a.665.665 0 0 0 .892-.331l2.05-4.638a.663.663 0 0 0-.331-.868 654.128 654.128 0 0 1-4.307-2.047c-5.817-2.83-10.75-2.644-14.523 3.502Z" fill="url(#a)"/><path d="M21.647 6.732c.216-.351.458-.762.663-1.083a.663.663 0 0 0-.225-.901l-4.307-2.65a.663.663 0 0 0-.914.225l-.636 1.063c-1.707 2.816-3.423 2.471-6.517.994L5.428 2.359a.662.662 0 0 0-.888.331L2.49 7.328a.663.663 0 0 0 .33.868c.902.424 2.697 1.269 4.307 2.047 5.814 2.806 10.747 2.62 14.52-3.511Z" fill="url(#b)"/><defs><linearGradient id="a" x1="21.369" y1="23.259" x2="8.34" y2="15.769" gradientUnits="userSpaceOnUse"><stop stop-color="#0050D3"/><stop offset=".94" stop-color="#007FFC"/><stop offset="1" stop-color="#0082FF"/></linearGradient><linearGradient id="b" x1="2.615" y1=".729" x2="15.645" y2="8.216" gradientUnits="userSpaceOnUse"><stop stop-color="#0050D3"/><stop offset=".94" stop-color="#007FFC"/><stop offset="1" stop-color="#0082FF"/></linearGradient></defs></svg>
\ No newline at end of file
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path d="M2.337 17.267c-.215.35-.457.758-.646 1.083a.663.663 0 0 0 .222.9l4.307 2.651a.663.663 0 0 0 .917-.225c.17-.288.391-.663.633-1.064 1.706-2.815 3.425-2.471 6.516-.993l4.27 2.03a.665.665 0 0 0 .892-.331l2.05-4.638a.663.663 0 0 0-.331-.868 654.128 654.128 0 0 1-4.307-2.047c-5.817-2.83-10.75-2.644-14.523 3.502Z"
+          fill="url(#confluence_l_a)"/>
+    <path d="M21.647 6.732c.216-.351.458-.762.663-1.083a.663.663 0 0 0-.225-.901l-4.307-2.65a.663.663 0 0 0-.914.225l-.636 1.063c-1.707 2.816-3.423 2.471-6.517.994L5.428 2.359a.662.662 0 0 0-.888.331L2.49 7.328a.663.663 0 0 0 .33.868c.902.424 2.697 1.269 4.307 2.047 5.814 2.806 10.747 2.62 14.52-3.511Z"
+          fill="url(#confluence_l_b)"/>
+    <defs>
+        <linearGradient id="confluence_l_a" x1="21.369" y1="23.259" x2="8.34" y2="15.769" gradientUnits="userSpaceOnUse">
```

Comment:
We need more unique ids like `intergalactic-icon-%name%-%version%`

Reply:
added generator in icon-transform-svg